src/main/resources/public/templates/imports.html

```diff
@@ -1,5 +1,24 @@
 <html>
 <head>
+
+
+    <style>
```

Comment:
maybe this should just be part of a style sheet we load on every page so we dont have to hard-code this everywhere